apps/web/src/services/iam/components/UserGroupTab.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { reactive, computed } from 'vue';
+
+import { PEmpty, PTab } from '@cloudforet/mirinae';
+import type { TabItem } from '@cloudforet/mirinae/types/navigation/tabs/tab/type';
+
+import { i18n } from '@/translations';
+
+import UserGroupTabDetail from '@/services/iam/components/UserGroupTabDetail.vue';
+import UserGroupTabUsers from '@/services/iam/components/UserGroupTabUsers.vue';
+import UserGroupTabWorkspaces from '@/services/iam/components/UserGroupTabWorkspaces.vue';
+import { USER_GROUP_TABS } from '@/services/iam/constants/user-group-constant';
+import { useUserGroupPageStore } from '@/services/iam/store/user-group-page-store';
+
+const userGroupPageStore = useUserGroupPageStore();
+const userGroupPageGetters = userGroupPageStore.getters;
+
+const singleItemTabState = reactive({
+    tabs: computed<TabItem[]>(() => ([
+        { label: i18n.t('IAM.USERGROUP.MAIN.DETAIL'), name: USER_GROUP_TABS.DETAIL },
+        { label: i18n.t('IAM.USERGROUP.MAIN.USER'), name: USER_GROUP_TABS.USER },
+        { label: i18n.t('IAM.USERGROUP.MAIN.WORKSPACE'), name: USER_GROUP_TABS.WORKSPACE },
+    ])),
+    activeTab: USER_GROUP_TABS.DETAIL,
+    // selectedIndex: computed(() => userPageState.selectedIndices[0]),
+    // selectedUserId: computed(() => userPageState.users[singleItemTabState.selectedIndex].user_id),
+});
+
+const multiItemTabState = reactive({
+    tabs: computed<TabItem[]>(() => ([
+        { label: i18n.t('IAM.USERGROUP.MAIN.DETAIL'), name: USER_GROUP_TABS.DETAIL }])),
+    activeTab: USER_GROUP_TABS.DETAIL,
+});
+</script>
+
+<template>
+    <section class="user-group-tab">
+        <p-tab v-if="userGroupPageGetters.selectedUsers.length === 1"
+               :tabs="singleItemTabState.tabs"
+               :active-tab.sync="singleItemTabState.activeTab"
+        >
+            <template #detail>
+                <user-group-tab-detail />
+            </template>
+            <template #user>
+                <user-group-tab-users />
+            </template>
+            <template #workspace>
+                <user-group-tab-workspaces />
+            </template>
+        </p-tab>
+        <p-tab v-else-if="userGroupPageGetters.selectedUsers.length > 1"
+               :tabs="multiItemTabState.tabs"
+               :active-tab.sync="multiItemTabState.activeTab"
+        >
+            <template #detail>
+                유저 그룹들
```

Comment:
nitpick : Even if it's a temporary markup, it's good to develop the habit of being careful with the wording, as it can be easily overlooked. If the code is still uncertain, it might be helpful to leave a mention like TODO.